administrator-manual/en/access.rst

```diff
@@ -30,16 +34,34 @@ as **root** and type the password chosen during |product| installation.
     The *unattended* install procedure sets the root password to the default
     ``Nethesis,1234``.
 
+.. _first-configuration-wizard-section:
+
+First configuration wizard
+==========================
+
+The first time **root** logs in the *First configuration wizard* procedure is
```

Comment:
Add a comma:
The first time **root** logs in**,**  the *First configuration wizard* procedure...